ethereum/core/src/main/java/tech/pegasys/pantheon/ethereum/mainnet/ProtocolSpecBuilder.java

```diff
@@ -272,17 +287,23 @@ TransactionProcessor apply(
   }
 
   public interface BlockProcessorBuilder {
+
     BlockProcessor apply(
         TransactionProcessor transactionProcessor,
         TransactionReceiptFactory transactionReceiptFactory,
         Wei blockReward,
         MiningBeneficiaryCalculator miningBeneficiaryCalculator);
   }
 
-  public interface BlockImporterBuilder<T> {
-    BlockImporter<T> apply(
+  public interface BlockValidatorBuilder<T> {
+
+    BlockValidator<T> apply(
         BlockHeaderValidator<T> blockHeaderValidator,
         BlockBodyValidator<T> blockBodyValidator,
         BlockProcessor blockProcessor);
   }
+
+  public interface BlockImporterBuilder<T> {
+    BlockImporter<T> apply(BlockValidator<T> blockValidator);
```

Comment:
nit: How come all the existing interfaces got an extra blank line added and the new one didn't?  (I like the no blank line version not that it really matters)

Reply:
threw a new blank line in for consistency.